draft-spaghetti-grow-nrtm-v4.xml

```diff
@@ -173,13 +185,14 @@
       </t>
       <t>
         In other words, either the mirror server guarantees that clients following the deltas have a correct and complete view, or MUST reinitialise, which will force clients to reinitialise as well.
+        !<-- SK: Should we define a procedure for that??? -->
```

Comment:
A procedure for reinitialising? It's the same procedure as for the first initialisation. The session ID rotation will force clients to reload.

Reply:
Hi Sasha... I think it was not clear on my side so let me rephrase it. What I mean is that in the draft is stated that the server is responsible to ensure that the clients have the correct & complete view of deltas. Should it be defined a process of detecting faults and a method to overcome those or we should leave it up to the server developers and focus only on the transport part?

Reply:
I think we focus more here on the transport part.

The scenario I had in mind is that there are some situations in IRRD where admins can discard the journal for a source. In that case, IRRD can't generate a contiguous set of deltas anymore for all changes that happened, therefore must reinitialise, forcing clients to reinitialise. Basically, a server either knows that it has complete history that it writes/wrote to delta files, or it might have missed things, and must reinitialise.